docs/features/agic-reconcile.md

```diff
@@ -5,17 +5,18 @@ When an Application Gateway is deployed through ARM template, a requirement is t
 To address the problem above, AGIC periodically checks if the latest gateway configuration is different from what it cached, and reconcile if needed to make gateway configuration is eventual correct.
 
 ## How to configure reconcile
-There are two ways to configure AGIC reconcile via helm
+There are two ways to configure AGIC reconcile via helm, and to use the new feature, make sure the AGIC version is at least at 1.2.0-rc1
+
 ### Configure inside helm values.yaml
-`reconcilePeriodSeconds=30`, it means AGIC checks the reconciling in every 30 seconds
+`reconcilePeriodSeconds: 30`, it means AGIC checks the reconciling in every 30 seconds
 
 ### Configure from helm command line
 Configure from helm install command(first time install) and helm upgrade command, helm version is v3
 ```bash
 # helm fresh install
-helm intall <releaseName> -f helm-config.yaml application-gateway-kubernetes-ingress/ingress-azure --set reconcilePeriodSeconds=30
+helm intall <releaseName> -f helm-config.yaml application-gateway-kubernetes-ingress/ingress-azure --version 1.2.0-rc1 --set reconcilePeriodSeconds=30 
```

Comment:
nit: extra space

Reply:
The command has a typo. "install" is missing the "s" character.

Reply:
thanks for pointing that out!